src/games/strategy/triplea/delegate/BattleDelegateOrdered.java

```diff
@@ -0,0 +1,114 @@
+package games.strategy.triplea.delegate;
+
+import java.util.Collection;
+
+import games.strategy.engine.data.Territory;
+import games.strategy.triplea.delegate.BattleDelegate;
+import games.strategy.triplea.delegate.IBattle;
+import games.strategy.triplea.delegate.IBattle.BattleType;
+import games.strategy.util.Match;
+
+/**
+ * This is to support the new maps added to world_war_ii_global which enforce strategic bombing first in the combat order @MapSupport
+ * @author simon
+ *
+ */
+public class BattleDelegateOrdered extends BattleDelegate {
+  // Fight all air and bombing raids which are supposed to be fought first. Order shouldn't matter because there won't normally be an attacker decision
+  private void stratBombing() {
```

Comment:
Method ordering should not be arbitrary. Have a read here: http://programmers.stackexchange.com/questions/199311/most-human-friendly-way-to-order-class-method-definitions, in this project please prefer to place methods below their first usage.